packages/synthetics-sdk-broken-links/test/unit/navigation_func.spec.ts

```diff
@@ -74,7 +74,7 @@ describe('GCM Synthetics Broken Links Navigation Functionality', async () => {
 
   after(async () => {
     // Close the browser after all tests
-    await browser.close();
+    if (browser) await browser.close();
```

Comment:
`browser ?? await bowser.close()`
or
maybe this works?
`await browser?.close()`

Reply:
browser && await browser.close();